cmd/apps/certmanager_app.go

```diff
@@ -95,6 +111,13 @@ func MakeInstallCertManager() *cobra.Command {
 			overrides["installCRDs"] = "true"
 		}
 
+		// Allow usage of lego behavior, see https://cert-manager.io/docs/usage/ingress/#optional-configuration
+		useAcme, _ := certManager.Flags().GetBool("acme")
+		if useAcme {
+			overrides["ingressShim.defaultIssuerName"] = "letsencrypt-production-issuer"
```

Comment:
I think this needs the same staging logix as the issuer as here, https://github.com/alexellis/arkade/pull/272/files#diff-90097b757fed057b84f0204d467ed1708d81bdf17fc9e0188fd9c22fcc04b577R219-R222

So the sataging issuer name is used if --staging is set?

Reply:
You're right, that's needed as well.

I'll change it

Reply:
Done.